Changelog.md

```diff
@@ -11,6 +11,8 @@ Bugfixes:
  * Code Generator: Fix length check when decoding malformed error data in catch clause.
  * SMTChecker: Fix false negatives in overriding modifiers and functions.
  * SMTChecker: Fix false negatives when analyzing external function calls.
+ * SMTChecker: Fix missing type constraints on block variable.
```

Comment:
```suggestion
 * SMTChecker: Fix missing type constraints for block variables.
```
I mean the `block.*` variables

Reply:
Should I formulate it as `block.*` or just "block variables", as in the suggestion?